src/app/static/src/app/store/filteredData/getters.ts

```diff
@@ -45,90 +45,5 @@ export const getters = {
             default:
                 return null;
         }
-    },
```

Comment:
All the logic here has moved into `choroplethData.ts`, almost unchanged, see one comment there.